dotnet/src/Functions/Functions.UnitTests/OpenAPI/RestApiOperationResponseConverterTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.SemanticKernel.Functions.OpenAPI.Model;
+using Xunit;
+
+namespace SemanticKernel.Functions.UnitTests.OpenAPI;
+public class RestApiOperationResponseConverterTests
+{
+    private readonly RestApiOperationResponseConverter _sut;
+
+    public RestApiOperationResponseConverterTests()
+    {
+        this._sut = new RestApiOperationResponseConverter();
+    }
+
+    [Fact]
+    public void ItShouldConvertStringContentToString()
+    {
+        //Arrange
+        var response = new RestApiOperationResponse("fake-content", "fake-content-type");
```

Comment:
Should the content type be `text/plain` here?

Reply:
The content type does not matter with the current implementation of the converter, which uses the type of the value of the `Content` property to decide on a conversion mechanism, as opposed to doing so based on the content type. However, the converter implementation can and will be changed to perform conversion based on the content type when and if needed.